app.go

```diff
@@ -107,15 +108,23 @@ type App struct {
 	getBytes func(s string) (b []byte)
 	// Converts byte slice to a string
 	getString func(b []byte) string
-	// Mounted and main apps
-	appList map[string]*App
 	// Hooks
 	hooks *Hooks
 	// Latest route & group
 	latestRoute *Route
 	latestGroup *Group
 	// TLS handler
 	tlsHandler *TLSHandler
+	// Mounted and main apps
```

Comment:
can you please encapsulate all mount relevant properties that are needed in a struct which is located in mount.go